src/views/preview/comment/comment.jsx

```diff
@@ -33,9 +43,39 @@ class Comment extends React.Component {
     }
 
     handleDelete () {
+        this.setState({deleting: true});
+    }
+
+    handleConfirmDelete () {
+        this.setState({deleting: false});
```

Comment:
is there a way we could wait for onDelete() to finish before setting state.deleting to false?

Reply:
That is a pending TODO comment in the preview reducer to add pending states to the API calls